File: frontend-app/src/domain/Alert.ts
interface Alert {
    id: string;
    title: string;
    category: string;
    subCategory: string;
    origin: string;
    assignedTo: string;
    status: string;
    creationTime: string;
    conclusionTime: string;
    description: string;
    resolution: string[];
    priority: string;
}

export default Alert;

File: backend_ml/src/domain/alert.py
class Alert:
    def __init__(self, id:str, priority:int, client:int, creation_time:str, category:str):
        self.id = id
        self.priority = priority
        self.client = client
        self.creation_time = creation_time
        self.category = category

    def __str__(self):
        return f"{self.client}"
        